### PR TITLE
fix: fix accessibility on storybook

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?
This PR fix some accessibility issues in sotrybook.

## How it works? 
Some storybook’ components have accessibility issues. The goal is to fix that.

- [x] Badge: Elements must have sufficient color contrast
- [x] Form elements must have labels
- [x] Price: Elements must have sufficient color contrast
- [x] Radio: Form elements must have labels
- [x] Select: Select element must have an accessible name
- [x] Textarea: Form elements must have labels
- [x] Carousel: aria-roledescription must be on elements with a semantic role
- [x] IconButton: Buttons must have discernible text
- [x] PriceRange: Elements must have sufficient color contrast
- [x] Table: Elements must have sufficient color contrast

### CODE REVIEW

File: packages/ui/.storybook/Welcome/welcome.css [hunk 1]
```diff
@@ -8,3 +8,7 @@
   height: 100vh;
   padding: 3rem;
 }
+
+.welcome__heading {
+  font-size: xx-large;
```
Comment (filipewl) [inline packages/ui/.storybook/Welcome/welcome.css:13]: TIL about `xx-large` and the other possible values, lol.
🆒

**Comment by filipewl** [comment]
Not sure it's in the scope of this task, but since I believe it's related, can you also add the assertion `toHaveNoIncompletes` from https://github.com/vtex/faststore/pull/1054 in the existing accessibility component tests? Maybe right after the ones that only have `toHaveNoViolations`.

**Review by filipewl — APPROVED** [review]
Niiice!

File: packages/ui/.storybook/Welcome/welcome.css (hunk 1 above)
Comment (saranicoly) [reply packages/ui/.storybook/Welcome/welcome.css:13]: Me too 🧐

**Review by saranicoly — APPROVED** [review]
Great job!

**Pull request merged**